### DISCUSSION TITLE
Can I ran test with different pylenium.json in cmd

### DISCUSSION THREAD

**Discussion by nguyet0310** [ElSnoMan/pyleniumio#264] [General]
I got a requirement, that run test in cmd, for Eg. if I run  python -m pytest tests/test_signin_page.py -> it will pick pylenium.json by default. but is it Ok to run python -m pytest tests/test_signin_page.py with another configuration file name pylenium_LT_Edge.json in cmd like, is there any thing like: python -m pytest tests/test_signin_page.py -c pylenium_LT_Edge.json

**Comment by ElSnoMan** [comment]
As of right now, no. You cannot pass in a config file to use. Instead, you would pass the CLI args for the config options you'd like to set.
With that said, this sounds like a great idea and I think we should add that as a feature so you can more easily configure pylenium. I'll add this as a new Issue 👍🏽

**Comment by ElSnoMan** [comment]
This was resolved in `Release 1.16.0` back in July. Release Notes here: https://github.com/ElSnoMan/pyleniumio/releases/tag/v1.16.0
